Mathlib/Order/OmegaCompletePartialOrder.lean

```diff
@@ -487,8 +598,21 @@ end CompleteLattice
 
 namespace CompleteLattice
 
-variable {α β : Type*} [OmegaCompletePartialOrder α] [CompleteLinearOrder β]
+variable [OmegaCompletePartialOrder α] [CompleteLinearOrder β] {f g : α → β}
+
+lemma ωScottContinuous.inf (hf : ωScottContinuous f) (hg : ωScottContinuous g) :
```

Comment:
What about sInf? Does that hold for nice set families?

Reply:
It may hold for finite sets, but I believe it's easy to find counterexamples for infinite sets.

Reply:
Abramsky and Jung 2.1.18 says that when `α` and `β` are directed complete partial orders then the space of Scott-continuous functions between them is also a directed complete partial order.

Reply:
Would be nice to have a version of `inf` for finite sets in that case, but feel free to leave it as a TODO for now

Reply:
I started thinking about this, but it opened up a lot more questions. I've left it as a TODO for now.